src/bin/console/console.cpp

```diff
@@ -521,7 +521,7 @@ int Console::visualize(const std::string& input) {
 
```

Comment:
Is it intended that percentage next to the arrows is `nan` sometimes? It would probably be nicer to have either nothing or an "unknown". Just not `nan` ;)

(can be seen e.g. at the top of the image from `projection` to `sort`)